pkg/operator/ceph/object/cosi/controller.go

```diff
@@ -152,14 +152,14 @@ func (r *ReconcileCephCOSIDriver) reconcile(request reconcile.Request) (reconcil
 	}
 
 	r.recorder.Eventf(cephCOSIDriver, kapiv1.EventTypeNormal, "Reconciling", "Reconciling Ceph COSI Driver %q", cephCOSIDriver.Name)
-	// Set the default CephCOSIDriver name if not already set
-	if cephCOSIDriver.Name == "" {
-		cephCOSIDriver.Name = CephCOSIDriverName
+	// The ceph-cosi-driver CRD must have the name "ceph-cosi-driver"
```

Comment:
Why? This seems like an unnecessary limitation when the operator can just know to query for a single CephCOSIDrivers in the operator namespace.

Reply:
Changed accordingly